vhd-format-lwt.opam

```diff
@@ -11,13 +11,13 @@ dev-repo: "git://github.com/mirage/ocaml-vhd"
 build: [[ "jbuilder" "build" "-p" name "-j" jobs ]]
 build-test: [[ "jbuilder" "runtest" ]]
 depends: [
-  "cstruct-lwt" {>= "1.9"}
+  "cstruct"
   "lwt" {>= "2.4.3"}
   "mirage-block"
   "mirage-types-lwt" {>= "3.0.0"}
+  "ounit"
```

Comment:
Shouldn't this be `{test}`?

Reply:
Nope, the real code also uses it for runtime assertions.

Reply:
```
vhd_format_lwt/patterns_lwt.ml
14:open OUnit
```